src/Network/Wai/SAML2.hs

```diff
@@ -40,13 +40,15 @@ import Network.Wai.SAML2.Assertion
 import Network.Wai.SAML2.Error
 
 import System.IO.Unsafe (unsafePerformIO)
+import Data.ByteString (ByteString)
 
 --------------------------------------------------------------------------------
 
 -- | Checks whether the request method of @request@ is @"POST"@.
 isPOST :: Request -> Bool 
 isPOST = (=="POST") . requestMethod
 
+
```

Comment:
Random empty line?

Reply:
it was a strategically placed empty line!